src/modules/booking/containers/booking-info-dialog/booking-info-dialog.tsx

```diff
@@ -0,0 +1,92 @@
+import { type FC } from 'react';
+import { faker } from '@faker-js/faker';
+import clsx from 'clsx';
+// components
+import { Avatar } from '@/modules/core/components/avatar';
+import { Typography } from '@/modules/core/components/typogrpahy';
+import { ServiceTag } from '@/modules/service/components/service-tag';
+import { Icon, type IconName } from '@/modules/core/components/icon';
+import { Button } from '@/modules/core/components/button';
+import { DialogBottom } from '@/modules/core/components/dialog-bottom';
+import { Dialog } from '@/modules/core/components/dialog';
+// hooks
+import { useDeviceType } from '@/modules/core/hooks/use-device-type';
+// assets
+import Girl from '@/assets/images/girl.png';
+
+import type { BookingInfoDialogProps } from './booking-info-dialog.interface';
+import styles from './booking-info-dialog.module.scss';
+
+const actions = [
+  {
+    id: 'call',
+    text: 'Call',
+    icon: 'phone',
+  },
+  {
+    id: 'reschedule',
+    text: 'Reschedule',
+    icon: 'time',
+  },
+  {
+    id: 'cancel',
+    text: 'Cancel reservation',
+    icon: 'close',
+    variant: 'danger',
+  },
+] satisfies {
+  id: string;
+  text: string;
+  icon?: IconName;
+  variant?: 'default' | 'danger';
+}[];
+
+export const BookingInfoDialog: FC<BookingInfoDialogProps> = ({
+  isOpen,
+  onOpenChange,
+}) => {
+  const deviceType = useDeviceType();
+  const DialogComponent = deviceType === 'mobile' ? DialogBottom : Dialog;
+
+  return (
+    <DialogComponent isOpen={isOpen} onOpenChange={onOpenChange}>
+      <div className={styles.root}>
+        <div className={styles.userInfo}>
+          <Avatar url={Girl.src} size='medium' />
+          <Typography className={styles.name} variant='subtitle'>
+            {faker.person.fullName()}
+          </Typography>
+          <Typography className={styles.email} variant='small'>
+            {faker.internet.email()}
+          </Typography>
+        </div>
+        <div className={styles.bookingInfo}>
+          <ServiceTag
+            // @ts-ignore
+            data={{ id: 'test', icon: 'nails', name: 'Evening makeup' }}
```

Comment:
We can import Service type here, so I thinkg we can create mock with this type so we don't need tot use ts-ignore